src/core/lib/surface/channel.h

```diff
@@ -85,4 +85,25 @@ void grpc_channel_internal_unref(grpc_channel* channel);
 grpc_compression_options grpc_channel_compression_options(
     const grpc_channel* channel);
 
+typedef struct registered_call {
```

Comment:
The definition of this struct can probably stay in the .cc file.  All we need here is a forward declaration of the struct.

Reply:
Done.